app/js/lib/speech/synthesis.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const p = Object.freeze({
+  // Properties
+  synthesis: Symbol('synthesis'),
+  supportsSynthesis: Symbol('supportsSynthesis'),
+  voice: Symbol('voice'),
+
+  // Methods
+  populateVoice: Symbol('populateVoice'),
+});
+
+export default class SpeechSynthesis {
+  constructor() {
+    const synthesis = window.speechSynthesis;
+
+    this[p.supportsSynthesis] = !!synthesis;
+
+    if (this[p.supportsSynthesis]) {
+      this[p.synthesis] = synthesis;
+      this[p.synthesis].onvoiceschanged = this[p.populateVoice].bind(this);
```

Comment:
Yeah I said "too", I know we need to wait for Chrome :)

Is it really an issue if it gets executed twice anyway ? It seems to me it's not an issue.

In chrome: called first time it does nothing because there are no voice, called second time it does what is expected.
In Firefox: called first time it does what is expected, never called a second time.

If the voices are actually changed on the device (unlikely) we'd like to call it once again anyway, right ?